xtas/tests/test_corenlp.py

```diff
@@ -92,7 +92,7 @@ def test_parse():
     assert_equal(corefs, {tuple(sorted(['John', 'himself']))})
 
 
-def test_multiple_sentences():
```

Comment:
It's also checking whether stanford_to_saf preserves the information it gets from CoreNLP.